scripts/generate-types-for-not-supported-ts-versions.mjs

```diff
@@ -15,11 +15,13 @@ limitations under the License.
 */
 
 import * as fs from 'node:fs';
-import { getTypeFilesInformation } from './shared/types-info.mjs';
+
 // generate warning when running with Node 18
 // (node:75278) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
 import packageJSON from '../package.json' assert { type: 'json' };
```

Comment:
**suggestion:** as @csouchet mentioned in person, `packageJSON` is not used in the script so it should be removed.
I guess it stayed by mistake after some utility codes were moved into `types-info.mjs` (where the variable is actually used). See 7356dab86860b6dfb8986a9aa193aca981b6b31e

Reply:
```suggestion
```